assets/xml/objects/gameplay_keep.xml

```diff
@@ -39,7 +39,7 @@
         <Texture Name="gameplay_keep_Tex_002E08" OutName="tex_002E08" Format="rgba16" Width="16" Height="16" Offset="0x2E08" />
         <Texture Name="gameplay_keep_Tex_003008" OutName="tex_003008" Format="i8" Width="8" Height="8" Offset="0x3008" />
         <Blob Name="gameplay_keep_Blob_003050" Size="0x20" Offset="0x3050" />
-        <DList Name="gDekuStickDL" Offset="0x32B0" />
+        <DList Name="gDekuStickDL" Offset="0x32B0" /> <!-- Supplies the wrong size to gsDPLoadTextureBlock, meaning it uses part of gLinkPauseChildJointTable as if it were a texture. -->
```

Comment:
I'd probably add a `@bug` tag.

Reply:
Agreed, I'll approve and merge when that's done.